modules/tensor_mechanics/include/userobjects/TensorMechanicsPlasticOrthotropic.h

```diff
@@ -0,0 +1,66 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef TENSORMECHANICSPLASTICORTHOTROPIC_H
+#define TENSORMECHANICSPLASTICORTHOTROPIC_H
+
+#include "TensorMechanicsPlasticIsotropicSD.h"
+
+class TensorMechanicsPlasticOrthotropic;
+
+template<>
+InputParameters validParams<TensorMechanicsPlasticOrthotropic>();
+
+/**
+ * Orthotropic plasticity model from Yoon (2013)
+ * the name of the paper is "Asymmetric yield function based on the
+ * stress invariants for pressure sensitive metals" published
+ * 4th December 2013.
+ * This model accounts for sensitivity in pressure, the
+ * strength differential effect and orthotropic behavior
+ * Yield_function = b*I1 + (J'2^{3/2} - c*J''3)^{1/3} - yield_strength
```

Comment:
if you enclose your formula in `\f$` it gets rendered as LaTeX in our online documentation.